.github/ISSUE_TEMPLATE/request-package-removal.yml

```diff
@@ -0,0 +1,38 @@
+---
+name: Request a package to be removed
+description: Request a package to be removed from the Solus repository. The package maybe unmaintained, replaced, deprecated or other reason.
```

Comment:
For the second sentence, maybe something like:

"The software may be unmaintained, replaced by another project, deprecated, or should be removed for some other reason."